### PR TITLE
Glacite Overlay (Oops)

### PR DESCRIPTION
**Pull request by f3shqt**
Adds Minecraft Cold Overlay to the Glacite Tunnels.
328560414-047dc57d-4437-4b78-a390-59b2c962b6ba

328580461-edae9813-e3e2-4100-b35b-e89b80c46038
The effect opacity changes with the cold value.

My bad for messing up the other pull request

### CODE REVIEW

File: src/main/java/de/hysky/skyblocker/config/configs/MiningConfig.java
Comment (AzureAaron) [inline src/main/java/de/hysky/skyblocker/config/configs/MiningConfig.java:93]: Move this bracket to the previous line

File: src/main/resources/assets/skyblocker/lang/en_us.json
Comment (AzureAaron) [inline src/main/resources/assets/skyblocker/lang/en_us.json:400]: Remove the `coldOverlay` reference in the translation string key. Also move it underneath the drill fuel so that its in alphabetical order with the other translation strings

File: src/main/java/de/hysky/skyblocker/config/configs/MiningConfig.java
Comment (AzureAaron) [inline src/main/java/de/hysky/skyblocker/config/configs/MiningConfig.java:95]: `coldOverlay`

**Review by AzureAaron — CHANGES_REQUESTED** [review]

**Comment by AzureAaron** [comment]
You forgot to move the strings down

File: src/main/resources/assets/skyblocker/lang/en_us.json
Comment (AzureAaron) [inline src/main/resources/assets/skyblocker/lang/en_us.json:417]: Should remove the `.enabled`, it isn't really needed

**Review by AzureAaron — CHANGES_REQUESTED** [review]

**Comment by kevinthegreat1** [comment]
Unfortunately conflicts with another pr, so this won't be included in bleeding edge for now. We'll have to wait until some of the other pr's are merged.

**Comment by AzureAaron** [comment]
Once the conflicts are resolved this can be merged

**Comment by kevinthegreat1** [comment]
Hey can you please run a `git pull master` and `git rebase master`?

**Review by kevinthegreat1 — APPROVED** [review]
Code looks good, but couldn't test.

**Pull request merged**
